algebird-test/src/test/scala/com/twitter/algebird/HyperLogLogTest.scala

```diff
@@ -216,6 +224,16 @@ class HyperLogLogTest extends WordSpec with Matchers {
       })
     }
 
+    "correctly downsize sparse HLL" in {
+      testDownsize(10)(10, 7)
+      testDownsize(10)(14, 4)
```

Comment:
can you test if you keep the bits the same it works, also if you increase the bits you get an exception.